back/package.json

```diff
@@ -14,7 +14,7 @@
     "migrate:dev": "ts-node -r dotenv/config prisma/migrate.ts",
     "check:migrations": "pg-validate-migrations ./prisma/migrations",
     "test": "jest",
-    "preintegration-tests": "npm run generate && npm run index-elastic-search:dev",
+    "preintegration-tests": "prisma db push && npm run generate && npm run index-elastic-search:dev",
```

Comment:
Ca veut dire que `prisma db push` sera joué à chaque `runtest` et non plus au `startcontainers`.
A voir si c'est problématique ou pas. J'aurais tendance à dire que non, mais il faudrait juste checker.

Pour rappel, un workflow courant est le suivant:
```bash
$ ./run.sh -u             // Call startcontainers - Je lance les containers
$ ./run.sh -r myTest      // Call runtest - Je run des tests en particulier
$ ./run.sh -r myTest
$ ./run.sh -r myTest
...
```

Reply:
Oui c'est vrai, mais je crois que `db push`est fait pour ce genre de use-case : https://www.prisma.io/docs/concepts/components/prisma-migrate/db-push. Je crois même que ça peut être avantageux d'executer `db push` à chaque `runtest`, si localement on itère sur le code/schema et qu'on relance les tests régulièrement.

Reply:
Comme db push inclu un generate, j'ai enlevé le duplicat dans la dernière version. On gagne qq dizaines de ms.